cli/loader.go

```diff
@@ -45,7 +43,7 @@ type LoaderConfig struct {
 }
 
 func NewLoader() Loader {
-	if isatty.IsTerminal(os.Stdout.Fd()) {
+	if isStdoutATerminal() {
```

Comment:
@keliramu I've changed the use the same function everywhere if stdout is a terminal